index.html

```diff
@@ -7663,64 +7659,276 @@ <h2>Acknowledgements</h2>
-Elliott, William Entriken, David Ezell, Nathan George, Reto Gmür, Ryan
-Grant, glauserr, Adrian Gropper, Joel Gustafson, Amy Guy, Lovesh
-Harchandani, Daniel Hardman, Dominique Hazael-Massieux, Jonathan Holt,
-David Hyland-Wood, Iso5786, Renato Iannella, Richard Ishida, Ian Jacobs,
-Anil John, Tom Jones, Rieks Joosten, Gregg Kellogg, Kevin, Eric Korb,
-David I. Lehn, Michael Lodder, Dave Longley, Christian Lundkvist, Jim
-Masloski, Pat McBennett, Adam C. Migus, Liam Missin, Alexander Mühle,
-Anthony Nadalin, Clare Nelson, Mircea Nistor, Grant Noble, Darrell
-O'Donnell, Nate Otto, Matt Peterson, Addison Phillips, Eric Prud'hommeaux,
-Liam Quin, Rajesh Rathnam, Drummond Reed, Yancy Ribbens, Justin Richer,
-Evstifeev Roman, RorschachRev, Steven Rowat, Pete Rowley, Markus
-Sabadello, Kristijan Sedlak, Tzviya Seigman, Reza Soltani, Manu Sporny,
-Orie Steele, Matt Stone, Oliver Terbu, Ted Thibodeau Jr, John Tibbetts,
-Mike Varley, Richard Varn, Heather Vescent, Christopher Lemmer Webber,
-Benjamin Young, Kaliya Young, Dmitri Zagidulin, and Brent Zundel.
+Abhishek Mahadevan Raju,
+Adam C. Migus,
+Adam Migus,
+Addison Phillips,
+Adrian Gropper,
+Aisp-GitHub,
+Alen Horvat,
+Alexander Mühle,
+AlexAndrei98,
+Allen Brown,
+Amy Guy,
+Andor Kesselman,
+Andres Paglayan,
+Andres Uribe,
+Andrew Hughes,
+Andrew Jones,
+Andrew Whitehead,
+Andy Miller,
+Anil John,
+Anthony Camilleri,
+Anthony Nadalin,
+ashimura,
+Benjamin Collins,
+Benjamin Goering,
+Benjamin Young,
+Bert Van Nuffelen,
+Bohdan Andriyiv,
+Brent Zundel,
+Brian Richter,
+Bruno Zimmermann,
+Bumblefudge,
+Butters,
+caribouW3,
+cdr,
+Chaoxinhu,
+Charles "Chaals" McCathieNevile,
+Charles E. Lehner,
+Chris Abernethy,
+Chris Buchanan,
+Christian Lundkvist,
+Christine Lemmer-Webber,
+Christoph Lange,
+Christopher Allen,
+Christopher Lemmer Webber,
+ckennedy422,
+Clare Nelson,
+confiks,
+Dan Brickley,
+Daniel Buchner,
+Daniel Burnett,
+Daniel Hardman,
+Darrell O'Donnell,
+Dave Longley,
+David Ammouial,
+David Chadwick,
+David Ezell,
+David Hyland-Wood,
+David I. Lehn,
+David Janes,
+David Waite,
+Denis Ah-Kang,
+Denisthemalice,
+Devin Rousso,
+Dmitri Zagidulin,
+Dominique Hazael-Massieux,
+Dostkamp,
+Drummond Reed,
+Elliott,
+Emmanuel,
+enuoCM,
+Eric,
+Eric Elliott,
+Eric Korb,
+Eric Prud'hommeaux,
+etaleo,
+Evstifeev Roman,
+Fabricio Gregorio,
+Filip Kolarik,
+Gabe Cohen,
+Ganesh Annan,
+George Aristy,
+glauserr,
+Golda Velez,
+Grace Huang,
+Grant Noble,
+Greg Bernstein,
+Gregg Kellogg,
+Haber,
+Heather Vescent,
+Henry Andrews,
+Henry Story,
+Ian B. Jacobs,
+Ilan,
+Isaac Henderson,
+isaackps,
+Iso5786,
+Ivan Herman,
+Jace Hensley,
+Jack Tanner,
+James Schoening,
+Jan Forster Cognizone,
+Jeff Burdges,
+Jeffrey Yasskin,
+Jim Masloski,
+Jim St.Clair,
+Joe Andrieu,
+Joel Gustafson,
+Joel Thorstensson,
+John Tibbetts,
+Jonathan Holt,
+José San Juan,
+Julien Fraichot,
+Justin Richer,
+Kaan Uzdoğan,
+Kaliya Young,
+Kazuyuki Ashimura,
+Ken Ebert,
+Kendall Weihe,
+Kerri Lemoie,
+Kevin,
```

Comment:
```suggestion
```

Likely a duplicate of one of the Kevins below.  Also, does not identify the contributor.